smf/internal-dns/config.toml

```diff
@@ -0,0 +1,22 @@
+[dropshot]
+bind_address = "[fd00:1234::4]:5353"
+request_body_max_bytes = 1048576
+
+[dns]
+bind_address = "[fd00:1234::4]:4753"
+
+[log]
+# Show log messages of this level and more severe
+level = "debug"
+
+# Example output to a terminal (with colors)
+mode = "stderr-terminal"
```

Comment:
I think we want these to be bunyan-formatted to a file.  I don't remember if there's already an option for that...if not, maybe /dev/stdout as a file?

Reply:
Sure.  The problem isn't that it won't be the right place.  The problem is that the plaintext formatting used for "terminal" is not machine-parseable the way bunyan is, which makes it quite a lot harder to count, filter, and otherwise process these log files for anything other than a human debugging a relatively simple problem.  We don't have to address it now though.

Reply:
I'm surprised by how easy that was - updating the logs to all be bunyan formatted (across all SMF services, including this one) was fairly easy, so I just went ahead and did it.